lessons/pydata/pandas/index.ipynb

```diff
@@ -0,0 +1,2872 @@
+{
```

Comment:
<p>Test nástroje ReviewNB...</p><p> </p><p>Kubův komentář se nespároval automaticky, škoda :-(</p><p> </p><p>Lambdu ze začátečníků umí? Jestli ne, tak možná přidat dvě věty?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Lambdu neumí. </p><p> </p>

Reply:
Už žádný další nový obsah prosím. Dejme to do dalších hodin, kde se to dá hned i ukázat na praktickém příkladu, i když to u lambdy moc nevidím, protože to není úplně triviální koncept a času je už tak málo.

Reply:
<p>Tak prosím lambdu vyhodit.</p>